src/main/search/SearchView.tsx

```
@@ -64,241 +69,580 @@ interface SearchViewProps {
+              break;
+            }
+          }
+
+          switch (resultsType) {
+            case SearchCategory.Spaces: {
+              return {
+                search: {
+                  ...prev.search,
+                  spaceResults: {
+                    ...fetchMoreResult.search.spaceResults,
+                    results: [...prev.search.spaceResults?.results, ...fetchMoreResult.search.spaceResults?.results],
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)